api/control/v1alpha1/types.go

```diff
@@ -49,19 +49,19 @@ type PlacementSpec struct {
 	// WantSingletonReportedState indicates that (a) the number of selected locations is intended
 	// to be 1 and (b) the reported state of each downsynced object should be returned back to
 	// the object in this space.
-	// When multiple Placement objects match the same workload object,
+	// When multiple BindingPolicy objects match the same workload object,
 	// the OR of these booleans rules.
 	// +optional
 	WantSingletonReportedState bool `json:"wantSingletonReportedState,omitempty"`
 }
 
-// PlacementStatus defines the observed state of Placement
-type PlacementStatus struct {
-	Conditions         []PlacementCondition `json:"conditions"`
-	ObservedGeneration int64                `json:"observedGeneration"`
+// BindingPolicyStatus defines the observed state of BindingPolicy
+type BindingPolicyStatus struct {
+	Conditions         []BindingPolicyCondition `json:"conditions"`
+	ObservedGeneration int64                    `json:"observedGeneration"`
 }
 
-// Placement is the Schema for the placementpolicies API
+// BindingPolicy defines in which ways the workloads ('what') and the destinations ('where') are bound together.
```

Comment:
+1 on writing a meaningful description here.
I think we have been saying "workload objects" rather than "workloads".

Reply:
Changed to "workload objects".